thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js

```diff
@@ -152,11 +196,21 @@ export default Route.extend({
    * @method model
    * @return {Object}
    */
-  model() {
+  model(transition) {
+    // Get duration data
+    const {
+      duration,
+      startDate,
+      endDate
+    } = transition;
+
     return RSVP.hash({
```

Comment:
no need to call `RSVP.hash` since `hash` was directly imported just use `hash()`

Reply:
done.